src/components/calcite-label/calcite-label.tsx

```diff
@@ -69,6 +69,9 @@ export class CalciteLabel {
   //
   //--------------------------------------------------------------------------
 
+  /**
+   * @internal
+   */
   @Event() calciteLabelFocus: EventEmitter<FocusRequest>;
```

Comment:
Same here ☝️.  This can probably be public unless I'm missing something.

Reply:
The reason I'd suggest hiding for now is that these focus events are not consistent across components. They'd still be available for use, but not in the doc for now. It'll be way easier to hide, revisit and make public after 1.0.0.

Reply:
`calcite-input` also has public focus/blur events. 🤔  

@macandcheese @driskull WDYT? Should we make these internal and replace them with more general `calciteFocus`/`calciteBlur` events?

Reply:
Yeah lets make them internal for now so we can make them consistent later